docs/src/neural_ode/neural_gde.md

```diff
@@ -293,21 +304,18 @@ st = st |> device
 For this task we will be using the `ADAM` optimizer with a learning rate of `0.01`.
 
 ```@example graphneuralode
-opt = Optimisers.ADAM(0.01f0)
-st_opt = Optimisers.setup(opt,ps)
+opt = ADAM(0.01f0)
+loss_clr(p) = loss(X, ytrain, train_mask, model, p, st)[1]
+optf = Optimization.OptimizationFunction((x,p)->loss_clr(x), Optimization.AutoZygote())
+optprob = Optimization.OptimizationProblem(optf, ps)
 ```
 
 ## Training Loop
 
-Finally, we use the package `Optimisers` to learn the parameters `ps`. We run the training loop for `epochs` number of iterations.
+Finally, we use the package `Optimization.solve` to learn the parameters `ps`. We run the training loop for `epochs` number of iterations.
 
 ```@example graphneuralode
-for _ in 1:epochs
-    (l,st), back = pullback(p->loss(X, ytrain, train_mask, model, p, st), ps)
-    gs = back((one(l), nothing))[1]
-    st_opt, ps = Optimisers.update(st_opt, ps, gs)
-    @show eval_loss_accuracy(X, y, val_mask, model, ps, st)
-end
+ps = Optimization.solve(optprob, opt, maxiters=1000, callback=callback).minimizer
```

Comment:
We don't need to return `ps` anymore

`Optimization.solve(optprob, opt, maxiters=1000, callback=callback)`